service-stub/src/main/java/com/iluwatar/servicestub/TickerServiceStub.java

```diff
@@ -0,0 +1,52 @@
+package com.iluwatar.servicestub;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+
+import yahoofinance.Stock;
+import yahoofinance.YahooFinance;
+
+public class TickerServiceStub implements TickerService{
```

Comment:
`TickerServiceStub` should have no dependency on Yahoo finance service (or real service). It should run in memory and should be in our control so that we can test finer tests very quickly.